types/react-native/globals.d.ts

```diff
@@ -272,3 +272,29 @@ declare class URLSearchParams {
     keys(): IterableIterator<string>;
     values(): IterableIterator<string>;
 }
+
+interface WebSocketMessageEvent extends Event {
+    data?: any;
```

Comment:
Should this perhaps be `unknown` nowadays?

Reply:
I will fix it in a new commit

Reply:
Changing from 'any' to 'unknown' requires TypeScript 3.0. The tslint configuration of the package requires TypeScript 2.9
To use unknown type I would have to specify TypeScript 3.0 for the whole file globals.d.ts.
I thought it may be a breaking change so did not do it

Reply:
Makes sense 👍